packages/rescript-relay-router/cli/RescriptRelayRouterCli__Commands.res

```diff
@@ -135,8 +134,6 @@ let generateRoutes = (~scaffoldAfter, ~deleteRemoved, ~config) => {
   let fileContents = `
 open RelayRouter__Internal__DeclarationsSupport
 
-external unsafe_toPrepareProps: 'any => prepareProps = "%identity"
-
```

Comment:
I replaced it with a safer:
```res
external unsafe_toPrepareProps: prepareProps<'a> => prepareProps<'b> = "%identity"
```
in RelayRouter__Internal__DeclarationsSupport.res